### PR TITLE
[insights-agent] update admission control

### PR DESCRIPTION
**Pull request by rbren**
**Why This PR?**
Update the admission controller sub-chart

**Changes**
Changes proposed in this pull request:

* Unpin the patch version of Fairwinds-managed charts
* Remove the requirements.lock in order to always update the fairwinds-managed patch versions

### CODE REVIEW

File: stable/insights-agent/requirements.lock [hunk 1]
```diff
@@ -1,15 +0,0 @@
-dependencies:
```
Comment (makoscafee) [inline stable/insights-agent/requirements.lock:1]: mmh why was this file deleted?

**Review by makoscafee — APPROVED** [review]

File: stable/insights-agent/requirements.lock (hunk 1 above)
Comment (rbren) [reply stable/insights-agent/requirements.lock:1]: We unpinned the versions so that we don't have to keep updating this chart every time a downstream has a patch.

**Review by sudermanjr — APPROVED** [review]
